spec/factories/lettings_log.rb

```diff
@@ -3,7 +3,10 @@
     created_by { FactoryBot.create(:user) }
     owning_organisation { created_by.organisation }
     managing_organisation { created_by.organisation }
+    created_at { Time.zone.today }
+    updated_at { Time.zone.today }
     trait :setup_completed do
+      startdate_today
```

Comment:
unrelated, noticed this while I was testing this ticket - startdate is part of a setup-completed lettings log so should be here